tests/plugins/test_rpmqa.py

```diff
@@ -49,16 +49,20 @@ def mock_logs_empty(cid, **kwargs):
     return ''
 
 
-def mock_logs_retry(cid, cache={}, **kwargs):
```

Comment:
Why this change in API? If needed it should be in separate commit. I don't see relation between fixing pylint warning and this change.

Reply:
IMO this was one of the rare cases where pylint actually reported a relevant warning. This method is not really usable more than once, because there is no good way to reset the cache. You would have to do something like this:
```python
retry_cache = {}
mock_logs_retry = partial(mock_logs_retry, cache=retry_cache)
# run a test
retry_cache.clear()
# run another test
retry_cache.clear()
# ...
```
The change essentially forces you to use clean cache every time you want to mock that method.
But if you think it should be in a separate commit that's ok with me.

Reply:
Yes please, use separate commit with description you provided here :)

That makes git archaeology much easier.